rm_drive_diagnostic_publisher:
  check_index:
    - "motors"
    - "rm_ecat"
  check_group:
    rm_ecat:
      - check_item: "warnings"
        regular_expression: ".*cycle time: (\\d+).*"
      - check_item: "counters"
        regular_expression: ".*cycle time: (\\d+).*"
    motors:
      - check_item: "positions"
        regular_expression: ".*cycle time: (\\d+).*"  # todo: avoid null
      - check_item: "velocity"
        regular_expression: ".*cycle time: (\\d+).*"
      - check_item: "warnings"
        regular_expression: ".*cycle time: (\\d+).*"